src/modules/FileLocksmith/FileLocksmithUI/Strings/en-us/Resources.resw

```diff
@@ -164,4 +164,7 @@
     <value>Administrator: File Locksmith</value>
     <comment>Title of the window when running as administrator.</comment>
   </data>
+  <data name="NotRunningAsAdminInfoBar.Message" xml:space="preserve">
+    <value>Some results might be missing because of insufficient permission. Restart as administrator to see them.</value>
```

Comment:
Personal opinion: this is still pretty vague, to me.

```suggestion
    <value>Run [File Locksmith] with admin rights to see more results.</value>
```

Reply:
This does not contain information about missing results becausing of insufficient permissions, maybe make it more clear?

Reply:
Would it make sense to use the text "Some results might be missing because of insufficient permission." and move the `restart as admin` button to the infobar as text button "Restart as administrator"?

Reply:
+1 on this. This is consistent with the OS and other PT modules.